noggin/tests/unit/conftest.py

```diff
@@ -178,34 +180,33 @@ def logged_in_dummy_user(client, dummy_user):
 
 @pytest.fixture
 def dummy_user_with_gpg_key(client, dummy_user):
-    ipa_admin.user_mod("dummy", fasgpgkeyid=["dummygpgkeyid"])
+    ipa_admin.user_mod(a_uid="dummy", fasgpgkeyid=["dummygpgkeyid"])
 
 
 @pytest.fixture
 def dummy_user_with_otp(client, logged_in_dummy_user):
     ipa = logged_in_dummy_user
     result = ipa.otptoken_add(
-        ipatokenowner="dummy",
-        ipatokenotpalgorithm='sha512',
-        description="dummy's token",
+        a_ipatokenuniqueid="dummy",
+        o_ipatokenowner="dummy",
+        o_ipatokenotpalgorithm='sha512',
+        o_description="dummy's token",
     )
-    token = OTPToken(result)
+    token = OTPToken(result['result'])
     yield token
     # Deletion needs to be done as admin to remove the last token
     try:
-        ipa_admin.otptoken_del(token.uniqueid)
+        ipa_admin.otptoken_del(result['result']['dn'][0])
```

Comment:
Are you sure? In `ClientMeta` it looks like the argument is the `uniqueid`, not the `dn`.

Reply:
Agreed.